avocado/utils/service.py

```diff
@@ -118,8 +118,9 @@ def method(cmd_result):
             # If service is stopped, exit_status is also not zero.
             # So, we can't use exit_status to check result.
             output = cmd_result.stdout.lower()
```

Comment:
Oh I'm sorry for not being clear. I meant the `output = cmd_result.stdout.lower()`, not the  `stderr`.